ontology/yaml/resources/fields/telemetry_fields.yaml

```diff
@@ -3129,6 +3129,10 @@ literals:
     fixed_min: 0.0
     flexible_max: 2000.0
 
+- electriccharge_sensor:
+    fixed_min: 0.0
+    flexible_max: 10000.0
```

Comment:
These values need to be in the STANDARD units as defined in the units.yaml. I am assuming you need to convert this to coulombs.